html/components/spinners.js

```diff
@@ -58,12 +73,13 @@ const spinners = () => {
     options.variant = spinnerContainer.getAttribute(
       'data-sprk-spinner-variant',
     );
+    options.role = spinnerContainer.getAttribute('data-sprk-spinner-role');
+    options.ariaValueText = spinnerContainer.getAttribute(
+      'data-sprk-aria-valuetext',
+    );
 
     spinnerContainer.addEventListener('click', (e) => {
-      if (
-        e.target.getAttribute('data-sprk-spinner') &&
```

Comment:
Why did we remove this line here? Did we change any functionality related to `data-sprk-spinner`? That we wouldn't need it anymore?

Reply:
@RCopeland  and I were looking at it the other day for a branching test, and we realized that it's not a useful check.

The spinner container (ie: button) will always have `data-sprk-spinner` otherwise this whole file is irrelevant.
This check was solely to make sure we're not adding spinner to something that's already spinning.